### PR TITLE
Hardcode some dark colors for theme idea "1b"

### PR DESCRIPTION
**Pull request by havocp**
This isn't intended to become the default, we'd want to
figure out a theme system that can affect these same
properties.

### CODE REVIEW

**Comment by havocp** [comment]
@fpliger this is for appmaker2 branch, not for mainline.

File: bokeh/charts/utils.py [hunk 1]
```diff
@@ -34,8 +34,11 @@
 #-----------------------------------------------------------------------------
 
 # TODO: (bev) this should go in a plotting utils one level up
+# TODO: is this used vs. the DEFAULT_PALETTE in _builder.py ?
```
Comment (fpliger) [inline bokeh/charts/utils.py]: IIRC it's used when the palette is not specific in `cycle_colors` while the one in _builder.py is used as a default chart argument when `cycle_colors` is called from a chart...

File: bokeh/charts/utils.py [hunk 2]
```diff
@@ -34,8 +34,11 @@
...
 _default_cycle_palette = [
-    "#f22c40", "#5ab738", "#407ee7", "#df5320", "#00ad9c", "#c33ff3"
+    "#b9bf3d", "#2cbdb9", "#8e4d9e",
+    # TODO replace the below with more from Sara
```
Comment (fpliger) [inline bokeh/charts/utils.py]: This should apply to the one in _builder as well

**Comment by fpliger** [comment]
Haven't tested locally. @havocp I can try it tomorrow morning

**Comment by havocp** [comment]
This isn't complete by any means by the way, it's just whatever was needed for a simple scatter plot example.

File: bokeh/charts/_builder.py [hunk 3]
```diff
@@ -23,7 +23,8 @@
 from ..models.ranges import Range
 from ..properties import Color, HasProps, Instance, Seq
 
-DEFAULT_PALETTE = ["#f22c40", "#5ab738", "#407ee7", "#df5320", "#00ad9c", "#c33ff3"]
+DEFAULT_PALETTE = [ "#b9bf3d", "#2cbdb9", "#8e4d9e" ]
```
Comment (birdsarah) [inline bokeh/charts/_builder.py]: Why just a 3 color palette here?

**Comment by birdsarah** [comment]
just curious: why not start bringing in variables for things like `#4d4d4d` that might make it clearer the places where you want similar colors. In particular I'm thinking variables like`background` rather than variables like `gray`. That way we can see how and where variables are reused.

File: bokeh/charts/_builder.py (hunk 3 above)
Comment (havocp) [inline bokeh/charts/_builder.py]: Sara didn't invent the others yet. :-)
She has more for her lighter theme.

**Comment by havocp** [comment]
Lack of variables and other good practices is because this is a hack for a demo. What we want to do for real is make all these values come from a pluggable theme.

**Comment by fpliger** [comment]
@havocp This looks good with the sliders example. Do you want to work more on this? Otherwise we can just merge and tweak a little next when the target examples are in better shape. What do you think?

**Comment by havocp** [comment]
yes, I'd say merge and then tweak a bit more as needed for the demo.

**Pull request merged**